### PR TITLE
Remove gap in wall tactics

### PR DESCRIPTION
**Pull request by Jason27chan**
## Description
See Title

## Associated Issue
Issue #1411 

## Design Documents
See issue

### Test Case 1
1. Run TestWall
2. Move Ball to various spots

Expected result: Wall does not error at any point

### Test Case 2
1. Put robots and wall in the following configuration
Start config
2. Run TestWall

Expected Result: No gap forms. The wall should block the direct shot

### CODE REVIEW

**Review by HussainGynai — APPROVED** [review]

**Comment by liquidmetal9015** [comment]
Does this fix the crash? Because the issue that it links links the issue that is the wall crashing.

File: soccer/gameplay/tactics/wall.py [hunk 1]
```diff
@@ -16,24 +16,25 @@ class State(enum.Enum):
         shot = 2
         scramble = 3
 
+    ##
+    # @param num_defenders: number of robots in the wall (default 3)
+    # @param curvature: 'curvature' (in radians) of the wall
```
Comment (kylestach) [inline soccer/gameplay/tactics/wall.py:21]: Could you explain what this actually means? Curvature should be in 1 / m

File: soccer/gameplay/tactics/wall.py [hunk 2]
```diff
@@ -196,3 +146,15 @@ def _update_wall(self):
             if self.has_subbehavior_with_name("robot" + str(i)):
                 behavior = self.subbehavior_with_name("robot" + str(i))
                 behavior.pos = self.calculate_destination(i)
+
+
+    ## Move behavior that prioritizes the center point of the wall
```
Comment (kylestach) [inline soccer/gameplay/tactics/wall.py:151]: What does "prioritizes the center point of the wall" mean?

**Review by kylestach — APPROVED** [review]
Tested, seems to fix gaps. It seems a little bit aggressive to cluster all of our robots in one spot, but it does what it says on the tin I guess.

LGTM

File: soccer/gameplay/tactics/wall.py (hunk 2 above)
Comment (Jason27chan) [reply soccer/gameplay/tactics/wall.py:151]: role requirements is determined by a robot's distance from the center part of the wall. The closest robot should always go to the center point of the wall

File: soccer/gameplay/tactics/wall.py (hunk 1 above)
Comment (Jason27chan) [reply soccer/gameplay/tactics/wall.py:21]: *shrug*

**Pull request merged**